Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml

```diff
@@ -94,6 +94,7 @@
       - id: GasAnalyzer
       - id: MedkitOxygenFilled
       - id: HolofanProjector
+      - id: DoorRemoteFirefight # Delta-V Re-added fire-fighting remote.
```

Comment:
why does `Delta-V` have a hyphen here but not anywhere else
`DeltaV` is the preferred spelling

Reply:
Because i did it half asleep and copied the comment below it without paying attention, my bad. Fixing it.